openfasoc/common/platforms/sky130hs/config.mk

```diff
@@ -90,8 +90,8 @@ export PLACE_DENSITY ?= 0.50
 export HOLD_BUF_CELL = sky130_fd_sc_hs__buf_1
```

Comment:
Are we using the `sky130hs`  for the digital sub blocks? 
I think the `sky130hd` makes more sense to improve the overall efficiency of the PMU

Reply:
Yeah, we originally used sky130hs for our whole design including digital sub-blocks, I can try to change them into sky130hd. Let me talk with Tuohang about the detail more.